src/main/java/org/isf/lab/manager/LabManager.java

```diff
@@ -265,6 +265,18 @@ public boolean newLaboratory2(Laboratory laboratory, List<LaboratoryRow> labRow)
 		}
 	}
 
+	/**
+	 * Inserts one Laboratory exam {@link Laboratory} (All Procedures)
+	 *
+	 * @param laboratory - the laboratory without result
+	 * @return <code>true</code> if the exam has been inserted, <code>false</code> otherwise
+	 * @throws OHServiceException
+	 */
+	public boolean newLaboratory3(Laboratory laboratory) throws OHServiceException {
+		setPatientConsistency(laboratory);
+		return ioOperations.newLabFirstProcedure(laboratory);
+	}
```

Comment:
why to add this method instead to modify and use the existing one `newLaboratory`?
This method is for FirstProcedure, so also the comment is wrong (All Procedures) and therefore is not clear what are you trying to do

Reply:
ok, as per discussion under OH2-183, the method should be named "newExamRequest", but the suggested status LABORATORY_ACTIVE=2 is not correct, LAB_ACTIVE (this the real field name) is for future use.
Please consider to better structure the model.

Reply:
@mwithi  have create the new method because all other method need exam result to be set. For this requiement we just need to set exam and the information of patient set. for the field LAB_ACTIVE, we have 3 case: LAB_ACTIVE=0  mince exam is delete LAB_ACTIVE = 1 mince exam is done and LAB_ACTIVE = 2 mince exam is requested. that is the logic

Reply:
the logic using LAB_ACTIVE as status field is wrong. Please read my comments under both OH2-183 and OHCS-84. Thanks